ui/static/ui/js/learning_resources.jsx

```diff
@@ -222,9 +222,9 @@ define('learning_resources', [
     VocabularyOption: VocabularyOption,
     LearningResourcePanel: LearningResourcePanel,
     loader: function (repoSlug, learningResourceId, container) {
+      React.unmountComponentAtNode(container);
```

Comment:
Can you please add a comment explaining what this fixes? it would be nice to be able to search for it in future :-)